tests/publiccloud/run_ltp.pm

```diff
@@ -156,34 +149,34 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    # LTP command line preparation
-    # The python3-paramiko is too old (2.4 on 15-SP6)
-    # The python311-paramiko is from SLE-Module-Python3-15-SP5-Updates which we have in PC tools image
-    zypper_call("in python311-paramiko python311-scp");
+    assert_script_run("cd kirk");
+    assert_script_run("python3.11 -m venv env311");
+    assert_script_run("source env311/bin/activate");
+    assert_script_run("pip3.11 install asyncssh msgpack");
 
     my $sut = ':user=' . $instance->username;
     $sut .= ':sudo=1';
     $sut .= ':key_file=$(realpath ' . $instance->provider->ssh_key . ')';
     $sut .= ':host=' . $instance->public_ip;
-    $sut .= ':reset_command=\'' . $reset_cmd . '\'';
+    $sut .= ':reset_cmd=\'' . $reset_cmd . '\'';
     $sut .= ':hostkey_policy=missing';
     $sut .= ':known_hosts=/dev/null';
 
-    my $cmd = 'python3.11 runltp-ng/runltp-ng ';
-    $cmd .= "--json-report=$root_dir/result.json ";
+    my $cmd = 'python3.11 kirk ';
+    $cmd .= "--framework ltp ";
     $cmd .= '--verbose ';
     $cmd .= '--exec-timeout=1200 ';
     $cmd .= '--suite-timeout=5400 ';
-    $cmd .= '--run-suite ' . get_required_var('LTP_COMMAND_FILE') . ' ';
+    $cmd .= '--run-suite ' . $ltp_command . ' ';
     $cmd .= '--skip-tests \'' . $skip_tests . '\' ' if $skip_tests;
     $cmd .= '--sut=ssh' . $sut . ' ';
     $cmd .= '--env ' . $env . ' ' if ($env);
+
     record_info('LTP START', 'Command launch');
-    assert_script_run($cmd, timeout => get_var('LTP_TIMEOUT', 30 * 60));
+    script_run($cmd, timeout => get_var('LTP_TIMEOUT', 30 * 60));
```

Comment:
kind question, why assertion here no more needed?

Reply:
because it always return `0`